tracing-subscriber/src/fmt/fmt_layer.rs

```diff
@@ -492,6 +492,22 @@ where
         }
     }
 
+    /// Sets whether or not the length of the [name] of the current thread is
+    /// tracked for padding.
+    ///
```

Comment:
I think the description of this method could probably do a somewhat better job of explaining what this controls. I don't know if it's super obvious what this does to users who haven't already seen the thread name padding behavior. Maybe we could say something like this:
```suggestion
    /// Sets whether or not the [name] of the current thread is padded to 
    /// the length of the longest thread name.
    ///
    /// If this is `true`, the formatter will track the length of the longest
    /// thread name that has been displayed, and pad any shorter thread
    /// names so that the thread name is always aligned with that many
    /// characters. If this is `false`, no additional whitespace will be added
    /// when displaying thread names.
```

We could also add an example of the padded and unpadded output to make it clearer to the user.